pyanaconda/payload/dnfpayload.py

```diff
@@ -1403,6 +1403,7 @@ class RepoMDMetaHash(object):
     def __init__(self, dnf_payload, repo):
         self._repoId = repo.id
         self._method = dnf_payload.data.method
+        self._ssl_verify = dnf_payload.is_ssl_verification_enabled
```

Comment:
This doesn't look correct either. It will use the value of `method.noverifyssl` for additional repositories again.

Reply:
This is now handled completely other way. I think it should be correct now.